cf/latest-snapshot/src/index.ts

```diff
@@ -40,6 +67,16 @@ export default {
 		const url = new URL(request.url);
 		const chain = (url.pathname.match(/\/latest\/(\w*)/) || ["undefined"])[1];
 
-		return await get_latest(env, chain);
+		// Disallow any other request method except GET, they are not sensible in the context
+		// of fetching a snapshot.
+		if (request.method !== 'GET') {
```

Comment:
Should we support `HEAD`? How does `aria2c` figure out how large the files are?

Reply:
`aria2c -x5` seems to be happy without it. It calls GET only.

```
12:03:32 PM GET /latest/calibnet/ 200
12:03:33 PM GET /latest/calibnet/ 206
12:03:33 PM GET /latest/calibnet/ 206
12:03:33 PM GET /latest/calibnet/ 206
12:03:33 PM GET /latest/calibnet/ 206
```